src/components/Admin/Admin.js

```diff
@@ -99,7 +99,31 @@ class Admin extends Component {
       textAlign: 'left',
       display: 'inline-block',
     };
-
+    const skillCountStyle = {
+      display: 'flex',
+      justifyContent: 'space-between',
+      marginTop: '50px',
+    };
+    const skillCountMobileViewStyle = {
+      width: '300px',
+      height: '210px',
+      marginBottom: '20px',
+      fontSize: '18px',
+      fontWeight: 'bold',
+      lineHeight: '2',
+    };
+    const totalSkillCount =
```

Comment:
Instead of declaring a variable, destruct the state object and use it, where it currently is.

Reply:
@fragm3 I have declared a variable because I have to use the same piece of code twice. If I don't do that complexity of render function check does not pass.